src/interface.ts

```diff
@@ -1,19 +1,20 @@
 export interface IBls {
+  implementation: Implementation;
   SecretKey: {
     fromBytes(bytes: Uint8Array): SecretKey;
     fromHex(hex: string): SecretKey;
     fromKeygen(ikm?: Uint8Array): SecretKey;
   };
   PublicKey: {
-    fromBytes(bytes: Uint8Array): PublicKey;
+    fromBytes(bytes: Uint8Array, type?: CoordType): PublicKey;
```

Comment:
It might be nice to have jsdoc strings in this file describing the default behavior for optional params